PoshRedisViewer/PoshRedisViewer/UILogic.fs

```diff
@@ -168,10 +168,24 @@ let setupViewsLogic multiplexer (views: Views) =
            |> ignore
        )
 
+
     views.ResultsListView
     |> ListView.addValueCopyOnRightClick id
     |> ListView.addValueCopyOnCopyHotKey id
-    |> ignore
+    |> fun resultsListView ->
+        resultsListView.add_KeyDown (fun keyDownEvent ->
+           let validIdx = resultsListView.Source <> null && resultsListView.SelectedItem > 0 && resultsListView.SelectedItem < resultsListView.Source.Count
+           
+           let selectedValue = if validIdx then resultsListView.Source.ToList()[resultsListView.SelectedItem] else ""
```

Comment:
I think it's better to not show anything if index is invalid?

Reply:
Yeah this was a WIP, I was still trying to figure out how to do an if statement ;) .

Reply:
Ok think that [7e82db8](https://github.com/En3Tho/PoshRedisViewer/pull/11/commits/7e82db852aaf29386994507d474546601db64958) is correct implementation.  Should no longer show blank.